src/responses/yuzu.json

```
@@ -45,7 +45,13 @@
-      "rec": { "reply": "For information on recommended settings and GPU drivers for yuzu, please refer to this page: <https://community.citra-emu.org/t/recommended-settings/319349>"}
+      "rec": { "reply": "For information on recommended settings and GPU drivers for yuzu, please refer to this page: <https://community.citra-emu.org/t/recommended-settings/319349>"},
+      "drivers": { "reply": "**Custom Drivers:** \nIf you have a device with a Qualcomm processor, you can explore the option of using custom drivers to potentially enhance performance and compatibility.\n\nPlease make sure to obtain the appropriate drivers for your specific device in the #android-drivers channel.\nFor Snapdragon 8 Gen1 or newer devices, you will need the Adreno 7xx drivers. For other Snapdragon devices, you will need the 6xx drivers."},
+      "ram": { "reply": "**Android RAM Requirements:** \nTo play most games on yuzu Android, a minimum of 8GB of RAM is currently required. While 6GB of RAM may work, it is only suitable for simple or small games."},
+      "powervr": { "reply": "**Unsupported PowerVR GPU** \nYour device is equipped with a PowerVR GPU, which lacks the necessary features for us to provide support."},
+      "directories": { "reply": "Here are the important directories in the `/Android/Data/org.yuzu-yuzu_emu` folder: \n\nFirmware: `/nand/system/Contents/registered` \nInstalled using the button in the app. \n\nSaves: `/nand/user/save/000000/profileID/gameID` \nYou can directly import saves as .zip files in the app, provided they are formatted correctly. This also works for importing saves from emulators such as Skyline. \n\nMods: `/load/gameID/` \nThere is currently no way to import mods through the app, you must install them manually using a file manager. \n\nInstalled Updates/DLC: `/nand/user/Contents/registered` \n\nDue to Android permission issues, most file managers cannot access the yuzu data directory. The easiest way to do so is to connect your phone to a PC and use its file manager. \nThere are also a number of third party file manager apps which can access the directory."},
+      "perf": { "reply": "yuzu Android is new and unoptimized. Performance issues, crashes, and graphical issues are to be expected. The current minimum/recommended specs are: \nA Snapdragon 865 SoC or newer. \nAt least 8GB of RAM(12GB for larger games like TOTK) \nA device running Android 11+ \n\nIf you cannot see the yuzu app in the Google Play Store, your device is unsupported."},
+      "bug": { "reply": "Please report any encountered issues here: \n<https://github.com/yuzu-emu/yuzu/issues/new/choose> \nIf your issue pertains to Yuzu Android, please mention it in the title."},
+      "ios": { "reply": "yuzu does not have an iOS version because Apple prohibits emulators on the app store, and there are no developers with an interest in developing on the platform."}
```

For the iOS command, I'd prefer if we just copied what the Citra bot already uses.

```suggestion
      "ios": { "reply": "yuzu does not have an official iOS version because iOS is a complicated operating system to develop for. We currently do not have any developers with an interest in developing for this platform."}
```

---

I thought about doing that, but didn't want to look lazy...